README.md

```diff
@@ -15,9 +15,14 @@ npm install react-chrome-redux
 
 ## Overview
 
-`react-chrome-redux` aims to do one simple thing for Chrome extensions: treat the background script as a single place to store a single source of truth redux Store.
+`react-chrome-redux` aims to do one simple thing for Chrome extensions: treat the background script as a place to store a single source of truth redux Store.
 
-All UI Components simply pull their react component state from this store on the background and dispatch actions to a store which proxies all actions to the background and receives state updates from the background.
+All UI Components follow the same basic flow:
+
+1. UI Component dispatches action to a Proxy Store.
+2. Proxy Store passes action to background script.
+3. Wrapped Store on the background script updates its state and sends back to UI Component.
```

Comment:
Having trouble with this wording. At this point in the documentation, "Wrapped Store" doesn't mean anything. Should we simplify this somehow?